server/raft.go

```diff
@@ -1989,16 +2055,25 @@ func (ar *appendEntryResponse) encode(b []byte) []byte {
 	return buf[:appendEntryResponseLen]
 }
 
+// Track all peers we may have ever seen to use an string interns for appendEntryResponse decoding.
+var peers sync.Map
```

Comment:
It looks like nothing ever gets removed from this `sync.Map`?

Reply:
For peers, yes, even on a large system like NGS with peer replacements would be like 40 entries.

Reply:
We could add a remove in there for peer remove entry..

Reply:
Added.